icalevents/icalparser.py

```diff
@@ -394,7 +412,8 @@ def parse_events(content, start=None, end=None, default_span=timedelta(days=7)):
                 exdate = "%04d%02d%02d" % (e.start.year, e.start.month, e.start.day)
                 if exdate not in exceptions:
                     found.append(e)
-    return found
+    # Filter out all events that are moved as indicated by the recurrence-id prop
+    return [event for event in found if e.sequence is None or not (event.uid, event.start, e.sequence) in recurrence_ids]
```

Comment:
I'd split this up for readability
```suggestion
    return [
        event for event in found 
        if e.sequence is None or not (event.uid, event.start, e.sequence) in recurrence_ids
    ]
```

Reply:
I get your point, but splitting up would also mean to do it in a for loop (except if you have a better idea?

Reply:
The version I suggested above still uses a list-comprehension, just split over mutliple rows to be more readable.